website/content/docs/plugins/creation/index.mdx

```diff
@@ -9,96 +9,83 @@ sidebar_title: Extending Packer
-  to implement for any given plugin.
+All you need to do to create a plugin is:
+  1. create an implementation of the desired interface, and
+  2. serve it using the server provided in the [packer-plugin-sdk](https://github.com/hashicorp/packer-plugin-sdk).
 
-- [`github.com/hashicorp/packer-plugin-sdk/plugin`](https://pkg.go.dev/github.com/hashicorp/packer-plugin-sdk/plugin) - Contains the code to serve
-  the plugin. This handles all the inter-process communication stuff.
+The core and the SDK handle all of the communication details inside the server.
 
-There are two steps involved in creating a plugin:
+Your plugin must use two packages from the SDK to implement the server and
+interfaces. You're encouraged to use whatever other packages you want in your
+plugin implementation. Because plugins are their own processes, there is no
+danger of colliding dependencies.
 
-1.  Implement the desired interface. For example, if you're building a builder
-    plugin, implement the [packer.Builder](https://pkg.go.dev/github.com/hashicorp/packer-plugin-sdk/packer#Builder) interface.
+- [`github.com/hashicorp/packer-plugin-sdk/packer`](https://pkg.go.dev/github.com/hashicorp/packer-plugin-sdk/packer) - Contains all the interfaces that you have to implement for any given plugin.
 
-2.  Serve the interface by calling the appropriate [plugin](https://pkg.go.dev/github.com/hashicorp/packer-plugin-sdk/plugin#Set.RegisterBuilder) serving method in
-    your main method.
-
-Basic examples are shown below. Note that if you can define a multi-plugin
-binary as it will allow to add more that one plugin per binary.
+- [`github.com/hashicorp/packer-plugin-sdk/plugin`](https://pkg.go.dev/github.com/hashicorp/packer-plugin-sdk/plugin) - Contains the code to serve the plugin. This handles all the inter-process communication.
 
+Basic examples of serving your component are shown below. Note that if you
+define a multi-component plugin, you can (but do not need to) add more than one
+component per plugin binary. The multi-component plugin is also compatible with
+download and installation via `packer init`, whereas the single-component plugin
+is not.
 
 <Tabs>
-<Tab heading="Multi-plugin Binary">
+<Tab heading="Multi-component Plugin (recommended) ">
 
 ```go
+// main.go
+
 import (
   "github.com/hashicorp/packer-plugin-sdk/plugin"
 )
 
-// Assume this implements packer.Builder
+// Assume this implements the packer.Builder interface
 type ExampleBuilder struct{}
 
-// Assume this implements packer.PostProcessor
+// Assume this implements the packer.PostProcessor interface
 type FooPostProcessor struct{}
 
-// Assume this implements packer.Provisioner
+// Assume this implements the packer.Provisioner interface
 type BarProvisioner struct{}
 
 func main() {
 	pps := plugin.NewSet()
 	pps.RegisterBuilder("example", new(ExampleBuilder))
+	pps.RegisterBuilder(plugin.DEFAULT_NAME, new(AnotherBuilder))
```

Comment:
Do you think we should add a comment here for the people that only read the code ?
```suggestion
  // When your `packer-plugin-coolcloud` defines a coolcloud builder
  // and you would like the plugin users to be able to use the
  // "coolcloud" builder instead of the "coolcloud-coolcloud" builder.
  // Use the `plugin.DEFAULT_NAME` keyword to skip the prefixing of 
  // the component with you plugin name
	pps.RegisterBuilder(plugin.DEFAULT_NAME, new(AnotherBuilder))
```

Reply:
( I am often one of these 😇  )